src/test/org/apache/nutch/api/AbstractNutchAPITestBase.java

```diff
@@ -160,6 +177,37 @@ private void initializeSSLProperties() {
     conf.set("restapi.auth.ssl.keypass", "password");
   }
 
+  private void loadCertificate() throws CertificateException, NoSuchAlgorithmException, KeyStoreException, IOException {
+    KeyStore ks = KeyStore.getInstance(KeyStore.getDefaultType());
+    char[] password = "testpassword".toCharArray();
+    String testTrustKeyStorePath = "src/test/testTrustKeyStore";
```

Comment:
Does this break Windows? I just give the path and I think it won't be a problem. Do I miss anything?